src/blog/templates/blog_search_tag.html

```diff
@@ -0,0 +1,97 @@
+{% extends "blog_base.html" %}
+{% load static %}
+
+{% block metatags %}
+    <meta name="description" content="{% block metadesc %}Selfpower.ro - Blog, Articole, Studii, Index{% endblock metadesc %}">
```

Comment:
Am o rugaminte daca poti:
- ca sa fi consitent cu denumirea view-ului `ListPostsByTag ` poti te rog redenumi fisierul asta din `blog_search_tag.html` in `list_post_by_tag.html`?
- te rog analizeaza si intelege un pic tag-urile <meta>: `description`, `title`, `keywords`
- vad ca nu am folosit meta title, doar meta og:title (og e de la OpenGraph adica pt Facebook share), poate trebuie adaugat si meta title pt google, poate citesti un pic. Sau poate nu, nu mai stiu de ce nu e inclus.
- pastreaza limba romana dar sa reflecte content-ul din pagina, de ex.:
 <meta name="description" content="{% block metadesc %}Selfpower.ro - Blog, Articole, Studii, Index{% endblock metadesc %}">
aici vad ca continutul vine dintr-un metadesc block, Dzeu stie de unde il iau, poate din model, chestia ar fi sa apara: Post-uri si articole despre `Fotbal`, `Educatie` sau cum se cheama tag-ul

la keywords: Selfpower, Blog, Articole, {% numele tag-ului %}

Reply:
_ca sa fi consitent cu denumirea view-ului ListPostsByTag poti te rog redenumi fisierul asta din blog_search_tag.html in list_post_by_tag.html?_
https://github.com/selfpower-ngo/selfpower.ro/commit/e596f29a14e12135efa31d63aa9f18bddcb59f9c
_te rog analizeaza si intelege un pic tag-urile : description, title, keywords
vad ca nu am folosit meta title, doar meta og:title (og e de la OpenGraph adica pt Facebook share), poate trebuie adaugat si meta title pt google, poate citesti un pic. Sau poate nu, nu mai stiu de ce nu e inclus._
aici vad ca continutul vine dintr-un metadesc block, Dzeu stie de unde il iau, poate din model, chestia ar fi sa apara: Post-uri si articole despre `Fotbal`, `Educatie` sau cum se cheama tag-ul_
https://github.com/selfpower-ngo/selfpower.ro/commit/42a4505ca513c1bcaa86a8b1392d14d4d5b859a8
la keywords: Selfpower, Blog, Articole, {% numele tag-ului _%}_
https://github.com/selfpower-ngo/selfpower.ro/commit/850b975bd061e25f48ffe46e938b52589deaa304